sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyAsyncClientTest.java

```diff
@@ -69,7 +69,10 @@ public void createKey(HttpClient httpClient, KeyServiceVersion serviceVersion) {
 
         createKeyRunner((keyToCreate) ->
             StepVerifier.create(keyAsyncClient.createKey(keyToCreate))
-                .assertNext(response -> assertKeyEquals(keyToCreate, response))
+                .assertNext(createdKey -> {
+                    assertKeyEquals(keyToCreate, createdKey);
+                    assertNotNull(createdKey.getProperties().getHsmPlatform());
```

Comment:
should we have a check for the actual value also ?

Reply:
I just wanted to make sure we deserialize properly but we can check for the value. It should be `0` by default from what I've seen.